stack/Cpp/Stack.cpp

```diff
@@ -0,0 +1,71 @@
+#include "Stack.h"
+#include<assert.h>
+
+template<class T>
+Node<T>::Node(T val)
+{
+	value = val;
+	next = 0;
+}
+
+template<class T>
+Node<T>::~Node()
+{
+}
+
+
+template<class T>
+void Stack<T>::push(T val)
+{
+	Node<T>* n = new Node<T>(val);
+	if (count == 0)
+		head = n;
+	else
+	{
+		n->next = head;
+		head = n;
+	}
+	count++;
+}
+
+template<class T>
+void Stack<T>::pop()
+{
+	assert(count != 0);
+	Node<T>* temp = head;
+	head = head->next;
+	delete temp;
+	count--;
+	
```

Comment:
Line contains following spacing inconsistencies:
- Trailing whitespaces.

*Origin: SpaceConsistencyBear, Section: `all.cpp`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpbn9qh2vv/stack/Cpp/Stack.cpp
+++ b/tmp/tmpbn9qh2vv/stack/Cpp/Stack.cpp
@@ -36,7 +36,7 @@
 	head = head->next;
 	delete temp;
 	count--;
-	
+
 }
 
 template<class T>
```